### ISSUE TITLE
WCAG 4.1.2,WCAG 1.3.1: ARIA hidden element must not contain focusable elements (#flightTools)

### ISSUE THREAD

**Issue by joshmccall221** [OfficeDev/office-ui-fabric-react#10739]
#### Issue
`ARIA hidden element must not contain focusable elements` ([aria-hidden-focus](https://dequeuniversity.com/rules/axe/3.3/aria-hidden-focus?application=msftAI))
#### Target application
[Home - Office UI Fabric](https://developer.microsoft.com/en-us/fabric#/controls/web/button)
#### Element path
#flightTools
#### Snippet
`<div id="flightTools" class="onedash-flightpicker" aria-hidden="true">`
#### How to fix
    Fix all of the following:
      Focusable content should have tabindex='-1' or be removed from the DOM
#### Environment
Chrome version 77.0.3865.90
This accessibility issue was found using Accessibility Insights for Web 2.10.0 (axe-core 3.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Comment by cliffkoh** [comment]
Hi @joshmccall221, when using accessibility insights, please test using https://aka.ms/fabricdemo which does not include content from the "host" page.

@Jahnp Is this something we should escalate to the right team?

**Comment by joshmccall221** [comment]
@cliffkoh same, if not more errors on fabric demo site. 

But the host site does not matter since the issue is on the  office-ui-fabric-react controls. 



https://developer.microsoft.com/en-us/fabric#/controls/web/button

![image](https://user-images.githubusercontent.com/7776611/66522059-649a9f80-eaa1-11e9-98a5-168d91f4147d.png)

https://fabricweb.z5.web.core.windows.net/oufr/7.46.0/#/examples/button

![image](https://user-images.githubusercontent.com/7776611/66522139-94e23e00-eaa1-11e9-93d6-a66e2726ae75.png)

Check out tool : 

https://accessibilityinsights.io/

Browser extension: 

![image](https://user-images.githubusercontent.com/7776611/66522504-6add4b80-eaa2-11e9-8404-611d89f6e56c.png)

**Comment by ecraig12345** [comment]
The specific `#flightTools` element is added by the developer.microsoft.com host site, not anything controlled by our code. We could potentially add code to hide it completely (since I'm pretty sure it serves no purpose for our particular website).

If Accessibility Insights is reporting issues with other Fabric controls, please file a separate GitHub issue about that.

**Comment by ecraig12345** [comment]
The specific `#flightTools` element is added by the developer.microsoft.com host site, not anything controlled by our code. We could potentially add code to hide it completely (since I'm pretty sure it serves no purpose for our particular website), but entirely removing it or implementing a proper fix would have to be done by the developer.microsoft.com team, not us.

If Accessibility Insights is reporting issues with other Fabric controls, please file a separate GitHub issue about that.

**Comment by ecraig12345** [comment]
Also, the `#flightTools` element is only shown to Microsoft internal users, not external users. So the impact is limited.

**Comment by ecraig12345** [comment]
I made a PR to hide the element (since we don't use it) and have also contacted the owning team to get the issue fixed properly.
